libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -601,22 +601,22 @@ void AC_AutoTune::update_gcs(uint8_t message_id) const
 {
     switch (message_id) {
     case AUTOTUNE_MESSAGE_STARTED:
-        gcs().send_text(MAV_SEVERITY_INFO,"AutoTune: Started");
+        GCS_SEND_TEXT(MAV_SEVERITY_INFO,"AutoTune: Started");
```

Comment:
This is a example where the whole function should be defined out.

Reply:
Scope creep :-)